### ISSUE TITLE
`>` symbols in css setting `settings.theme` got escaped to `&gt;`

### ISSUE THREAD

**Issue by tete1030** [brookhong/Surfingkeys#1484]
## Prelude
* Could you find answer in below documents?

    1. [FAQ](https://github.com/brookhong/Surfingkeys/wiki/FAQ) / [常见问题](https://github.com/brookhong/Surfingkeys/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98)
    1. [README](https://github.com/brookhong/Surfingkeys/blob/master/README.md) / [读我](https://github.com/brookhong/Surfingkeys/blob/master/README_CN.md)

No

* Have you searched your problem in [issues](https://github.com/brookhong/Surfingkeys/issues)?

Yes

## Error details

SurfingKeys: 0.9.74

Browser: Chrome 91.0.4472.106 macOS 11.4

URL: All pages

In Surfingkeys Settings:
![image](https://user-images.githubusercontent.com/4312115/122640927-a879ef00-d134-11eb-94dc-62afa1b91301.png)

In DevTools:
![image](https://user-images.githubusercontent.com/4312115/122640905-808a8b80-d134-11eb-82d6-b30cedef619b.png)

## Context

Set theme through `setting.theme` and observe actually inserted css from dev tools

**Comment by tete1030** [comment]
Related: #767, #1247

**Comment by tete1030** [comment]
https://github.com/brookhong/Surfingkeys/blob/de0a580d50f27ccd1acf7d514a6752131ccf79c6/pages/front.js#L254

I believe `.textContent = message.userSettings.theme` would be sufficient, special characters would not be interpreted.

**Comment by tete1030** [comment]
Same goes for https://github.com/brookhong/Surfingkeys/blob/de0a580d50f27ccd1acf7d514a6752131ccf79c6/pages/front.js#L336

**Comment by brookhong** [comment]
Yes, you have found this cause, but this change is not allowed since the reviewers from AMO will reject this, you can just replace the `>` with space.

**Comment by tete1030** [comment]
If you mean they will reject code that inserts content to document without sanitizing, can we add a correct sanitizer for `.textContent`, which does nothing? I think this will indicate to the reviewers clearly that no symbol requires escape in this situation.
